viz/charts/linkerd-viz/README.md

```diff
@@ -164,6 +164,7 @@ Kubernetes: `>=1.13.0-0`
 | tapInjector.image.registry | string | `"ghcr.io/linkerd"` | Docker registry for the tapInjector instance |
 | tapInjector.image.tag | string | `"linkerdVersionValue"` | Docker image tag for the tapInjector instance |
 | tapInjector.keyPEM | string | `""` | Certificate key for the tapInjector. If not provided then Helm will generate one. |
+| tapInjector.logLevel | string | defaultLogLevel | log level of the tapInjector |
```

Comment:
I wonder if we should change this so that we show the actual default value. In other places where we have default log levels, we use the log level name instead of `defaulLogLevel`. WDYT?
```suggestion
| tapInjector.logLevel | string | `"info"` | log level of the tapInjector |
```

Reply:
This file is auto-generated through `bin/helm-docs` which uses the comments on the `values.yaml` file. Anyways,I think it's fine to refer to other fields in the same `values.yaml` file. This goes in line with #5694, after which a change in `defaultLogLevel` will cascade down to all the other log levels, so I think it's important to reflect that in the default value.

Reply:
Ahh, I gotcha. This makes total sense. I didn't realize this file was auto-generated. Thanks for the explanation!